server/src/main/java/org/cloudfoundry/identity/uaa/util/beans/DbUtils.java

```diff
@@ -0,0 +1,123 @@
+package org.cloudfoundry.identity.uaa.util.beans;
+
```

Comment:
can you please check for sonar smells
https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=1950&id=cloudfoundry-identity-parent&open=AYIdJm8AY6EzI4nYYplF

Reply:
Sure we will check this thanks.

Reply:
We fixed the security warnings and critical code smells. For DbUtils class we were able to fix 5 major code smells.